civihr_default_theme/assets/sass/partials/_panel.scss

```diff
@@ -88,6 +88,15 @@
     position: relative;
 }
 
+.chr_panel--without-background {
+   background: none;
+   box-shadow: none;
+
+   .chr_search-result__header--border-darker {
```

Comment:
As I talked with @AkA84, as `chr_search-result__header--border-darker` is related with `.chr_panel--without-background`, it's better to be nested.